src/datasets/inspect.py

```diff
@@ -271,9 +271,12 @@ def get_dataset_split_names(
     )
     if builder.info.splits is None:
         try:
+            download_config.use_auth_token = use_auth_token
```

Comment:
by the way, I'm not sure if I should do this, or if `get_dataset_split_names` should be called with both `use_auth_token` and `download_config` parameters (seems overly complicated)

Reply:
Simply passing `download_config` is ok :)